common/src/main/java/com/wynntils/models/ingredients/IngredientInfoRegistry.java

```diff
@@ -111,7 +111,11 @@ public IngredientInfo deserialize(JsonElement jsonElement, Type jsonType, JsonDe
 
             Optional<String> internalNameOpt = Optional.ofNullable(internalName);
 
-            int tier = JsonUtils.getNullableJsonInt(json, "tier");
+            // The tier field was renamed to rarity in API v3.3, but it really does not make sense, so
+            // I'll keep parsing both tier and rarity for now (as they may change it back)
+            int tier = json.has("tier")
+                    ? JsonUtils.getNullableJsonInt(json, "tier")
+                    : JsonUtils.getNullableJsonInt(json, "rarity");
```

Comment:
They mentioned changing this to back to tier so I don't think you need this anymore

Reply:
Let's keep both for now; if they stabilize we can clean this up later. 

I've come to deeply mistrust the Wynncraft API. :-( They have had a hard time getting their act together on that...